packages/dashboard/src/components/resizablePanes/index.tsx

```diff
@@ -45,6 +53,8 @@ export const ResizablePanes: FC<ResizablePanesProps> = ({ leftPane, centerPane,
   /** Current widths of the three panes, in px */
   const [leftPaneWidth, setLeftPaneWidth] = useState(DEFAULT_SIDE_PANE_WIDTH);
   const [rightPaneWidth, setRightPaneWidth] = useState(DEFAULT_SIDE_PANE_WIDTH);
+  const [leftPaneCollapse, setLeftPaneCollapse] = useState(LEFT_PANE_COLLAPSED);
```

Comment:
curious , why do you need local state ? when you already have the redux value ?

Reply:
removed the redux store.